packages/nouns-webapp/src/components/PropLotList/PropLotList.module.css

```diff
@@ -0,0 +1,205 @@
+.heading {
+  font-family: 'Londrina Solid';
+  font-size: 40px;
+  margin: 0 !important;
+}
```

Comment:
no love for tailwind? :)

Reply:
Ha yes, I've become a fan! This is here as it was just a copy paste of the Ideas component so I could get a rough example up